### DISCUSSION TITLE
Update @sap-ux/ui5-middleware-fe-mockserver from 2.1.98 to 2.2.28 some functions failed

### DISCUSSION THREAD

**Discussion by Esmee-Zhang** [SAP/open-ux-odata#801] [Q&A]
Hi, I noticed that when I update the version from 2.1.98 to 2.2.28, some functions did not work.
For example, in order to click a customized button on the object page, I use Test Recorder generated action to click it. It works well on 2.1.98, but it seems the test cannot find the button by this method in version 2.2.98. Is there other general method to locate the customized button on the Object page?

**Comment by nlunets** [comment]
That sounds like an OPA question and not a mockserver one, can you clarify hjow the mockserver is related here ?
